src/main/java/nl/tudelft/ti2206/group9/gui/GameScreen.java

```diff
@@ -49,13 +50,18 @@
 	private static SubScene worldScene;
 	/** The overlayscene. */
 	private static SubScene overlayScene;
+
 	/** The ExternalTicker to be used. */
 	private static ExternalTicker extTicker;
 	/** Indicate whether the game is running. */
 	private static boolean running;
 	/** The primarystage. */
 	private static Stage primaryStage;
-	/** The Pause popup. */
+
+	/** The AudioPlayer to be used for background music. */
+    static AudioPlayer audioPlayer = new AudioPlayer("src/main/resources"
```

Comment:
*Auto-generated comment*

    Variable 'audioPlayer' must be private and have accessor methods.